components/Carousel/Carousel.js

```diff
@@ -1,11 +1,75 @@
 class Carousel {
+    constructor(carousel){
+        this.element = carousel;
+        this.leftButton = document.querySelector('.left-button');
```

Comment:
For `leftButton`, `rightButton`, and `carouselImgs`, the best choice for grabbing those elements may not be `document`. The documentation for `querySelector` says it is also viable for the parent node of the elements you are grabbing.